test/fastly/dictionary_test.rb

```diff
@@ -2,13 +2,13 @@
 
 describe Fastly::Dictionary do
 
-  let(:client)     { Fastly.new(user: 'test@example.com', password: 'password') }
+  let(:client)     { Fastly.new(api_key: 'notasecrettestkey', user: 'test@example.com', password: 'password') }
```

Comment:
Do we still need to pass the `user`, `password` params now they've been deprecated?

Reply:
I won't repeat the comment, but it applies to the other files in this PR where a similar construct is used.